scripts/antares_test_utils.py

```diff
@@ -14,16 +13,17 @@ def list_directories(directory):
             dir_list.append(x)
     return dir_list
 
-def list_studies(directory):
+def find_studies_in_batch_dir(batch_name):
+    batch_directory = Path(batch_name).resolve()
     studies = []
-    dir_path = Path(directory)
+    dir_path = Path(batch_directory)
```

Comment:
dir_path == batch_directory ?

Reply:
I simplified this. 
Thanks for noticing it.